Makefile

```
@@ -4,19 +4,21 @@ RTL := $(shell python3 misc/convert_filelist.py Makefile rtl/rtl.f)
 # TOP := screen_driver_tb
 TOP := alu_add_tb
 
+INCLUDE := -I$(shell yosys-config --datdir)
```

```suggestion
YOSYS_DATDIR := $(shell yosys-config --datdir)
```
I wanted to keep generic include paths inside the .f files. I don't want to create confusion that all include paths belong in the Makefile. This is simply a exception